### DISCUSSION TITLE
HttpError: Unprocessable Entity: "Could not clone: was submitted too quickly"

### DISCUSSION THREAD

**Discussion by PogorzelskaPaulina** [community/community#28843] [Apps, API and Webhooks]
I'm getting **HttpError: Unprocessable Entity: "Could not clone: was submitted too quickly"** on **POST https://api.github.com/repos/{template_owner}/{template_repo}/generate** endpoint. 

Endpoint was working with the same body before. Any idea ?

**Comment by danielmeppiel** [comment]
Hi Paulina,

I believe you are hitting the so called "Secondary rate limits" of GitHub API. These are limits on content creation to avoid abuse of the API. They are currently not documented, but you can read a bit more about them [here](https://github.com/cli/cli/issues/4801). 

The only way forward I can suggest is to wait for 1 hour whenever you hit the "submitted too quickly" error.
